content/en/docs/multi-center/modules/icu/chartevents.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "chartevents"
+linktitle: "chartevents"
+weight: 3
+date: 2023-09-18
+description: >
+  Charted items during the ICU stay include numeric vital signs within respiratory, routine, general, and hemodynamic categories
+---
+
+**Links to:**
+
+* patients on `subject_id`
+* admissions on `hadm_id`
+* icustays on `stay_id`
+* d_items on `itemid`
+
+
+# Table columns
+
+Name | Postgres Data type
+---- | --------
+subject\_id | INTEGER
+hadm\_id | INTEGER
+stay\_id | INTEGER
+caregiver_id | INTEGER
+charttime | TIMESTAMP(0)
+storetime | TIMESTAMP(0)
+itemid | INTEGER
+value | VARCHAR(200)
+valuenum | DOUBLE PRECISION
+valueuom | VARCHAR(20)
+warning | SMALLINT
+
+### `subject_id`, `hadm_id`
+
+Identifiers which specify the patient: `subject_id` is unique to a patient, `hadm_id` is unique to a patient hospital stay.
+
+### `stay_id`
+
+`stay_id` is an integer identifier which uniquely identifies each ICU admission within a hospital admission. Each hospital admission `hadm_id` must have at least one ICU admission `stay_id` but may include more than one. A `stay_id` groups all ICU admissions within 24 hours of each other.
+
+### `caregiver_id`
```

Comment:
is this intentional? *caregivers* table is not present

Reply:
removed!